src/App.js

```diff
@@ -29,10 +40,12 @@ class App extends Component {
     return (
       <main className = 'App'>
         <Header currentState={this.state.currentMovie}/>
-        {!this.state.currentMovie &&
+        {this.state.error &&
+         <h1>Sorry no movies found 🤷‍♂️</h1>}
+        {!this.state.currentMovie && !this.state.error &&
           <Area className='area' movies={this.state.movies} details={this.showMovieDetails}/>
         }
-        {this.state.currentMovie &&
+        {this.state.currentMovie && !this.state.error &&
```

Comment:
Since this will prevent the Details background styling from displaying we should add some styling for our error container.

Reply:
Yes! Or - consider building a reusable Error component? Could have a little error image, and maybe a button to go back home and try again? Up to you!